tests/scripts/thread-cert/Cert_8_1_01_Commissioning.py

```diff
@@ -65,7 +65,7 @@ def test(self):
 
         self.nodes[JOINER].interface_up()
         self.nodes[JOINER].joiner_start('openthread')
-        time.sleep(10)
+        time.sleep(15)
```

Comment:
I was hoping to see log messages like:

- `Joiner::HandleDtlsTransmit`
- `Commissioner::HandleDtlsReceive`

It might also be useful to see the mbedtls logs to get more context on what DTLS handshake messages are being sent/received.